utils/metadata.py

```diff
@@ -50,16 +54,22 @@ def generate(
     """
     log.info("Generating metadata.")
 
+    if retain_nifti == output_nrrd:
```

Comment:
And it does. On line 160 of `parse_config.py`, I swap the `retain_nifti` config to `False` if `output_nrrd` is `True`, because that's the expected behavior of dcm2niix.

I also place that logic here because it is relevant to the parameters of this function in particular. Future utilization then only needs to look at this function to be understood as the logic and functionality are explicitly retained into one unit.